server/tinylicious/.devcontainer/Dockerfile

```diff
@@ -1,7 +1,12 @@
 # Copyright (c) Microsoft Corporation and contributors. All rights reserved.
 # Licensed under the MIT License.
 
-FROM node:14.19.1
+# Override this by setting the REGISTRY argument when calling docker build
+# to use a different registry than docker hub
+# docker build . REGISTRY=<custom registry>
+ARG REGISTRY=docker.io
+
+FROM ${REGISTRY}/node:14
```

Comment:
Hey @hedasilv 
Thanks for popping in for a review :) 
I've asked about that minor version difference, and it seems that previously there was an issue with specific versions but it has long been solved? @tylerbutler might be able to chime in on this

Reply:
The minor version specification previously (i.e. the .19.1) was to ensure downloading the latest Node version with the latest security vulnerability patches. It would be counter-productive to install an earlier version of Node.js with known security vulnerabilities in the name of using a secure container registry... @ms-avocado does the `node:14` image get updated to the latest minor version automatically? And how do we know that we are pulling the latest version if it is not tagged? In other words, if I install node:14 image, it is cached locally, so even if it is updated without a new tag, I will never be able to pull it without manually clearing cache, correct?

Reply:
I'm not sure which minor MCR is using, but docker.io resolves to 14.19.3
I believe docker's default behavior when minor version is untagged is to pull the latest available version
but yes the caching problem exists. We could avoid that by adding the `--pull` option to docker build, ensuring it checks the registry for updates before deciding to use the local cache